### PR TITLE
[WIP] Added a top UIStackView to MessageInputBar

### PR DESCRIPTION
**Pull request by nathantannar4**
This is to support features like @mention and a typing indicator. Updated the Example to show changes.

There are some **breaking changes** but only with the backgroundView of the MessageInputBar to support difference background colors behind the top stack view and main content

### CODE REVIEW

**Comment by SD10** [comment]
I don't mind adding another `UIStackView` but something doesn't feel right about having the typing indicator inside of the `MessageInputBar`. I really don't want these two components coupled 😕

The interaction between the keyboard/message input bar and the collectionview is already really painful. I'm not sure I want to add more state management to the part of the library.

Let's leave this open and see what others think.

File: Sources/MessageInputBar.swift [hunk 1]
```diff
@@ -27,27 +27,39 @@ import UIKit
...
     
     /// When set to true, the blurView in the background is shown and the backgroundColor is set to .clear. Default is FALSE
     open var isTranslucent: Bool = false {
         didSet {
-            blurView.isHidden = !isTranslucent
-            backgroundColor = isTranslucent ? .clear : .white
+            if isTranslucent && blurView.superview == nil {
+                backgroundView.addSubview(blurView)
+                blurView.fillSuperview()
+            } else if !isTranslucent {
```
Comment (SD10) [inline Sources/MessageInputBar.swift]: It's probably best just to add this once if they set`isTranslucent` to `true`

File: Sources/MessageInputBar.swift [hunk 2]
```diff
@@ -89,6 +110,8 @@ open class MessageInputBar: UIView {
         return view
     }()
     
+    open let typingIndicatorLabel = UILabel()
```
Comment (SD10) [inline Sources/MessageInputBar.swift]: We need to remove this as we discussed

**Review by SD10 — CHANGES_REQUESTED** [review]
Let's just leave the top stack view and I think this is good :P

**Comment by nathantannar4** [comment]
Sorry I accidentally committed a merge to this PR. Its for autocomplete

**Comment by SD10** [comment]
No worries 😃 I know the `MessageInputBar` is your jam but I really need to limit what goes into `v0.9.0`. This is also ~500 lines of code now so it can become difficult for me to review. Let's try and push enhancements of the `MessageInputBar` until after `v0.9.0`. Bugfixes are welcome any time

**Comment by ohazar** [comment]
Hello, what is the status of this change? Is it going to be added or canceled? Ticket looks "closed" but its label is "in progress".

**Pull request closed without merging**